interpreter/CMakeLists.txt

```diff
@@ -525,4 +525,9 @@ set(CPPINTEROP_INCLUDE_DIRS
 #---Set InterOp to build on Cling, this can be toggled to use Clang-REPL-------------------------------------------------------
 set(CPPINTEROP_USE_CLING ON BOOL "Use Cling as backend")
 
+#---Disable testing on InterOp if ROOT's testing is OFF (InterOp tests are enabled by default) -------------------------------------------------------
+if(NOT testing)
+set(CPPINTEROP_ENABLE_TESTING OFF)
```

Comment:
```suggestion
  set(CPPINTEROP_ENABLE_TESTING OFF)
```

Reply:
Don’t we just need set in the cache with force?